integration_tests/pages/barcode/createNewContact.ts

```diff
@@ -28,4 +30,8 @@ export default class CreateNewContactPage extends Page {
   prisonIdField = (): PageElement => cy.get('#prisonId')
 
   submitButton = (): PageElement => cy.get('#create-new-contact-form button')
+
+  static goToPage = (): CreateNewContactPage => FindRecipientByPrisonNumberPage.goToPage().happyPath()
+
+  happyPath = (): ReviewRecipientsPage => this.submitWithValidValues()
```

Comment:
Each page has a static `goToPage` function to get to the page, and a `happyPath` to get to the next page. This way all pages don't have to know about how to get to its previous page which prevents the duplication of the test setup logic.